examples/006_filter_composite_data_example.py

```diff
@@ -4,9 +4,15 @@
 Filter result data by different criteria
 ----------------------------------------
 
-This example show how data filtering can be used for custom postprocessing of
+This example shows how data filtering can be used for custom postprocessing of
```

Comment:
```suggestion
This example shows how you can use data filtering for custom postprocessing of
```
Use active voice when possible.